app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -46,13 +46,23 @@ def loa2_or_above?
   end
 
   def identity_not_verified?
-    UserDecorator.new(current_user).identity_not_verified?
+    !current_user.verified?
```

Comment:
TAL at https://github.com/18F/identity-idp/pull/314 where I have axed this method entirely. I can rebase from that if you approve.